packages/block-editor/src/components/block-settings-menu/index.js

```diff
@@ -106,18 +104,8 @@ export function BlockSettingsMenu( { clientIds, onSelect } ) {
 									onToggle={ onClose }
 								/>
 							) }
-							<ReusableBlockConvertButton
-								clientIds={ clientIds }
-								onToggle={ onClose }
-							/>
 							<_BlockSettingsMenuPluginsExtension.Slot fillProps={ { clientIds, onClose } } />
 							<div className="editor-block-settings-menu__separator block-editor-block-settings-menu__separator" />
-							{ count === 1 && (
```

Comment:
I guess with this change, "Remove from Reusable Blocks" is moved to before the horizontal line in the menu, ungrouped from Remove Block. I don't see it being much an issue (grouped with reusable convert button vs. grouped with remove block button), but worth pointing out.

Before|After
---|---
![Before](https://user-images.githubusercontent.com/1779930/54946161-dc421c00-4f0d-11e9-8b88-f449529cfaec.png)|![After](https://user-images.githubusercontent.com/1779930/54946127-cb91a600-4f0d-11e9-8d57-d9dcc9bb91a3.png)